target.json

```diff
@@ -2,7 +2,7 @@
   "name": "nrf51dk-armcc",
   "version": "0.0.8",
   "inherits": {
-    "nordic-nrf51822-32k-armcc": "*"
+    "nordic-nrf51822-armcc": "*"
```

Comment:
We definitely could, but this might introduce a maintenance problem because then every time we make any major updates to nordic-nrf51822-armcc then we have to modify at least 3 targets i.e. nrf51dk-armcc mkit-armcc and bbc-microbit-armcc.